### DISCUSSION TITLE
tailwindcss worked in vite dev mode,but it was crashed in build

### DISCUSSION THREAD

**Discussion by shlroland** [tailwindlabs/tailwindcss#3260] [Help]
when i execute `vite` cli that is vite dev mode, tailwindcss can work currently.but it cannt bundle in css file when execute `vite build`. so what should i do?

OS: Macos 11 Big Sur
Node: 14.15.1
yarn：1.22.10
vite: 1.0.0-rc.13

**Comment by simonswiss** [comment]
Hey!

Are you able to share a repo with a reproduction of the issue so we can take a look at it?

**Reply by shlroland** [reply]
[rol-ui-vue3](https://github.com/shlroland/rol-ui-vue3) its here. i used vite and tailwind to write docs for my ui library.
